Description:  change to 1024x768 resolution
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2020-04-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Flawed/data/settings.ini
+++ b/Flawed/data/settings.ini
@@ -1,3 +1,3 @@
-hres 800
-vres 600
+hres 1024
+vres 768
 seed 42
